packages/core/src/mixins/__tests__/anchored.test.js

```diff
@@ -153,53 +153,49 @@ it('renders below when anchor placed near top of viewport', () => {
     expect(boxWrapper.prop('style').top).toBe(50 + ANCHOR_SIZE);
 });
 
-it('renders and aligns to the center line of anchor', () => {
+it('aligns to the center of anchor if space is enough on both left and right', () => {
     const anchorWrapper = mount(<Anchor top={20} left={100} />, { attachTo: anchorRoot });
     const boxWrapper = mount(
         <AnchoredBoxTop anchor={anchorWrapper.instance()} />,
         { attachTo: boxRoot }
     ).find(Box);
 
-    expect(
-        // The center line of <Box>
-        boxWrapper.prop('style').left + (BOX_SIZE / 2)
-    ).toBe(
-        // The center line of <Anchor>
-        100 + (ANCHOR_SIZE / 2)
-    );
-});
+    expect(boxWrapper.prop('style').left)
+        // 100 + (ANCHOR_SIZE / 2) - (BOX_SIZE / 2)
+        .toBe(60);
 
+    // Center-aligned component doesn't specify arrow style.
+    expect(boxWrapper.find(Box).prop('arrowStyle')).toEqual({});
+});
 
-it('renders on the left when anchor placed on the right of viewport', () => {
+it('aligns to the right side of anchor if space is not enough on right side', () => {
```

Comment:
`if space is not enough on left side` ?

Reply:
This should be closer to current behavior of `anchored()` here.

When your anchor is placed near the right edge of viewport, so you cannot align the “anchored component” to the center of anchor.

In this scenario the “anchored component” is placed (on top or below) the anchor, with its right edge aligned to the right edge of anchor.

An “anchored component” is always placed on top or below the anchor in current implementation.

Reply:
I misunderstood the anchored component is "expanding" to the right side XD

Reply:
That's why I changed the text because I was also confused when reading that XD